source/basic.tex

```diff
@@ -2680,6 +2687,14 @@
 they both declare names with external linkage.
 \end{itemize}
 \begin{note}
+An \grammarterm{alias-declaration}\iref{dcl.typedef},
```

Comment:
How about `using type = struct A{};`?

Reply:
It's true that an *alias-declaration* can **contain** a declaration of an entity, but I think it's clear here that it is the direct effect of the top-most declaration that is in question.

I'm not sure I understand the second question, although note that the paragraph begins by "enabling" names for linkage for purposes.

Reply:
I think `alias-declaration` and `declaration specified with typedef` should have the same treatment since they are merely different ways to declare the identifier to be a typedef-name. In line 2690, we just say `alias-declaration` but do not mention the declaration with typedef specifier, it's the second question.